par_test.go

```diff
@@ -314,87 +313,145 @@ func BenchmarkReduce(b *testing.B) {
 	})
 }
 
-var result bool
-
 func BenchmarkAny(b *testing.B) {
-	rand.Seed(time.Now().Unix())
+	rand.Seed(1)
```

Comment:
The reason I used time was to provide some variation, otherwise you only ever get the results of one position in the benchmark. This does make it random, but it still provide good relative results, thoughts?

Reply:
The needle is picked based on how far along in the `n < b.N` iteration we are, so as long as we get enough repetition, the index should be fairly uniform. As a side-note, the reason why it's OK to rely on `b.N` like this (as opposed to, say, using it as a size which would cause the benchmark time to increase infinitely) is that as `b.N` increases, the selected indices become *more* uniform, bringing us closer to the average, rather than increasing it, whenever `b.N` is increased.